index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+ 
+</head>
+<body>
+    <p> "Hello World, we are software developers!" </p>
```

Comment:
I kindly suggest that you use the ["h1"](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements) tag  for this main heading instead of using a "p" tag. For headings or heading texts, the use of header tags e.g h1, h2, h3 ...h6 is better recommended and more appropriate to use. Please I suggest that you change the "p" tag to an "h1" tag, especially to enable Google and other search engines recognize the title for this webpage.

Reply:
-  [ ] Great work @geekelo 👏🏾